src/wcf/4.8/windowsservercore-2009/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# escape=`
+
+ARG REPO=mcr.microsoft.com/dotnet/framework/aspnet
+FROM $REPO:4.8-windowsservercore-2009
+
+# Install Windows components required for WCF service hosted on IIS
+RUN powershell -Command `
+        $ErrorActionPreference = 'Stop'; `
+        $ProgressPreference = 'SilentlyContinue'; `
+        `
+        Add-WindowsFeature NET-WCF-TCP-Activation45; `
```

Comment:
Since we are already using DISM in the runtime Dockerfiles, it would be more consistent if we used it here and in the aspnet Dockerfiles instead of pwsh.

```
DISM /Online /Quiet /Enable-Feature /FeatureName:WCF-TCP-Activation45
```

Reply:
This works but it also requires an additional command to disable the IIS-WebServerManagementTools feature that gets installed as a result of installing IIS-WebServerRole in order to keep parity with the features that were being installed previously with the usage of PowerShell.